tests/pyfunc/test_model_export_with_class_and_artifacts.py

```diff
@@ -1552,7 +1552,7 @@ def predict(self, context, model_input: List[str], params=None) -> List[str]:
     assert pyfunc_model.metadata.get_input_schema().to_dict() == [
         {"type": "string", "required": True}
     ]
-    pd.testing.assert_frame_equal(pyfunc_model.predict(["a", "b"]), pd.DataFrame(["a", "b"]))
+    assert pyfunc_model.predict(["a", "b"]) == ["a", "b"]
```

Comment:
Wondering what's changed here, it's no longer a pandas dataframe output?

Reply:
The return type is `List[str]` (line 1542), which converts the return value into a list of strings.

Reply:
This test fails if `TestModel` is defined in the top level on master.